css/style.css

```diff
@@ -0,0 +1,10 @@
+@media screen and (min-device-width: 1200px) and (max-device-width: 1600px) {
+  #navbar {
+    float: left;
+    position: fixed;
+  }
+}
+
```

Comment:
I think its better to remove unwanted  lines

Reply:
> I think its better to remove unwanted lines

- [ ] Kindly provide a reason as to why you are requiring this change